src/elements/types/ExportDefaultDeclaration.js

```diff
@@ -0,0 +1,24 @@
+import ModuleDeclaration from '../ModuleDeclaration';
+
+export default class ExportDefaultDeclaration extends ModuleDeclaration {
+    constructor(childNodes) {
+        super('ExportDefaultDeclaration', childNodes);
+    }
+
+    _acceptChildren(children) {
+        children.passToken('Keyword', 'export');
+        children.skipNonCode();
+        children.passToken('Keyword', 'default');
+        children.skipNonCode();
+        let declaration = children.passExpression();  // or declaration
```

Comment:
I think this is a very custom thing and we don't need to put it to `elementAssert`. Just get `currentElement` and check for `VariableDeclaration` or `FunctionDeclaration` and otherwise call `passExpression()`.